pandas/tools/pivot.py

```diff
@@ -170,6 +170,9 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     if len(index) == 0 and len(columns) > 0:
         table = table.T
 
+    if isinstance(table, Series):
```

Comment:
can you see if you can find the actual root cause here?

e.g. may be as simple as making sure that the column/index seletors are list-like and not scalars